### ISSUE TITLE
upper/lower case issue for letter "wxyz" on Alphabet USB Overlay

### ISSUE THREAD

**Issue by dglaude** [adafruit/Adafruit_IntelliKeys#3]
### Operating System

Windows 10

### Arduino IDE version

? or n/a

### Board

Feather RP2040 USB Host

### ArduinoCore version

? or n/a

### TinyUSB Library version

? or n/a

### Sketch as ATTACHED TXT

.

### Compiled Log as ATTACHED TXT

.

### What happened ?

I am not an IntelliKeys expert, and I only manadged to make it work today with the 0.2.1 UF2.
(I was less lucky with a previous release).

I'll have to hack the code and/or hack overlay as I am an AZERTY user, but right now I just switch my Windows PC to English/UK keyboard.
Also notice that my InteliKey is from UK and not from US (I don't think there are a lot of difference, or maybe none, just support contact).

Anyway, I tested the overlay with those writing on it:
```
IntelliKeys USB (British English)
ALPHABET USB OVERLAY
ITEM NO. OL - USB - UK - AL/BW

Copyright (C) 2002 IntelliTools, Inc.
SUPPORT: info@inclusive.co.uk
tel: +44 (0)1457 819790
```

The overlay is all lowercase from a to z in alphabetical order.
When I press the intellikeys USB, I get all uppercase letter, except for the last row where I get "wxyz".

`.,?!ABCDEFGHIJKLMNOPQRSTUVwxyz`

I am not sure about the official driver and if it should be uppercase or lowercase, but it is wrong to have that mix.

Regards

### How to reproduce ?

Insert the Alphabet overlay
Press from a to z.

### Debug Log

Here are the event I see on the serial port:
```
membrane [22, 04] = 1
membrane [22, 04] = 0

membrane [21, 07] = 1
membrane [21, 07] = 0

membrane [21, 10] = 1
membrane [22, 10] = 1
membrane [21, 10] = 0
membrane [22, 10] = 0

membrane [21, 13] = 1
membrane [22, 13] = 1
membrane [21, 13] = 0
membrane [22, 13] = 0
```

**Comment by dglaude** [comment]
![PXL_20230812_163619462](https://github.com/adafruit/Adafruit_IntelliKeys/assets/19435932/39c855b5-fdb4-4bc2-8cca-a11ef1a6242c)

Here is the overlay I am talking about.

**Comment by dglaude** [comment]
I have tested the all lower solution #5 using the artifact produced by my PR (I could not test #4 as my PR fail on some code formatting).

All lower is the solution I prefer, the one that make more sense to me and match the printed character on the physical layer.

PS: I have a separate problem with "." that does not seems to send anything (but I ear a beep). But that problem is present both with the official `0.2.1 UF2` and with the artifact UF2 from #5 .
I don't understand why I did not detect that earlier.
I also don't understand by reading the code why this line is not doing it: https://github.com/adafruit/Adafruit_IntelliKeys/blob/dd9482e5657f0f20626a04068ce4baf9a0c3ff8a/src/IKOverlay.cpp#L739C1-L740C63
But that should be a separate issue I guess... I don't think I introduced this.

**Comment by hathach** [comment]
> All lower is the solution I prefer, the one that make more sense to me and match the printed character on the physical layer.

I agree, all lower case is better and merged. Thank you for your fix in #5.

> PS: I have a separate problem with "." that does not seems to send anything (but I ear a beep). But that problem is present both with the official 0.2.1 UF2 and with the artifact UF2 from https://github.com/adafruit/Adafruit_IntelliKeys/pull/5 .
I don't understand why I did not detect that earlier.
I also don't understand by reading the code why this line is not doing it: https://github.com/adafruit/Adafruit_IntelliKeys/blob/dd9482e5657f0f20626a04068ce4baf9a0c3ff8a/src/IKOverlay.cpp#L739C1-L740C63
But that should be a separate issue I guess... I don't think I introduced this.

that is weird, I put my Ikeys to storage, I will try to pull it out to test later on. Meanwhile, could you test with other overlay which also has `Period` to see if the code actually work on other membrane switch. And yes, please open an seperate issue for period if possible. It is also nice if you could provide the membrane log as this issue as well, just to see if the feedback is actually for that Period (mine Ikeys has quite a lots of dead membrane).

Thank you for both this issue and the fix.
